docs/openshift/decommission/decommission-openshift-resources.md

```diff
@@ -183,4 +183,120 @@ parameters after `--all` within your selected project (namespace).
     Make sure to backup any important data or configurations before executing this
     command to prevent accidental data loss.
 
+Please check that all resources currently being used by your project by running
+`oc get all`, as shown below:
+
+    oc get all
+    NAME                        TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                               AGE
+    service/modelmesh-serving   ClusterIP   None         <none>        8033/TCP,8008/TCP,8443/TCP,2112/TCP   7m4s
+
+!!! note "Important Note"
+    The last remaining service, i.e., `service/modelmesh-serving`, shown when running
+    the `oc get all` command, is a **REQUIRED** resource, and so you don't need
+    to clean it up.
+
+## Use ColdFront to reduce the Storage Quota to Zero
+
+Each allocation, whether requested or approved, will be billed based on the
+**pay-as-you-go** model. The exception is for **Storage quotas**, where the cost
+is determined by [your requested and approved allocation values](../../get-started/allocation/allocation-details.md#pi-and-manager-allocation-view-of-openshift-resource-allocation)
+to reserve storage from the total NESE storage pool. For **NERC-OCP (OpenShift)**
+Resource Allocations, storage quotas are specified by the "OpenShift Request
+on Storage Quota (GB)" and "OpenShift Limit on Ephemeral Storage Quota (GB)"
+allocation attributes.
+
+Even if you have deleted all Persistent Volume Claims (PVC) in your OpenShift project.
+It is very essential to adjust the approved values for your NERC-OCP (OpenShift)
+resource allocations to zero (0) otherwise you still be incurring a charge for
```

Comment:
you -> you will

Reply:
@joachimweyl : I was referring as user already have deleted the PVC following previous step they need to perform the mentioned process. Let me know if still you want to change it?

Reply:
grammatically "you still be" does not work, if you have another way you would like to say it that is fine.

Reply:
Got it - changing it now.